public/video-ui/src/reducers/publishedVideoReducer.js

```diff
@@ -3,9 +3,33 @@ import { blankVideoData } from '../constants/blankVideoData';
 export default function video(state = null, action) {
   switch (action.type) {
     case 'PUBLISHED_VIDEO_GET_RECEIVE':
-      return action.publishedVideo
-        ? Object.assign({}, blankVideoData, action.publishedVideo)
-        : false;
+      if (action.publishedVideo) {
+        let video = Object.assign({}, blankVideoData, action.publishedVideo);
+        /*
+        this is necessary to make sure that if the old expiryDate field already exists in the db,
+        the new expiry value is initialised to the saved one. This won't be necessary
+        once we remove the old field
+        */
+        const newExpiryDate =
+          video.contentChangeDetails.expiry &&
+          video.contentChangeDetails.expiry.date;
+        if (!newExpiryDate && video.expiryDate) {
+          video = Object.assign({}, video, {
```

Comment:
could we just `return` rather than mutate `video` (also means we can use a `const` instead of a `let` above)